tests/0061-consumer_lag.cpp

```diff
@@ -131,8 +131,7 @@ static void do_test_consumer_lag (void) {
   if (conf->set("event_cb", &stats, errstr) != RdKafka::Conf::CONF_OK)
     Test::Fail("set event_cb failed: " + errstr);
   Test::conf_set(conf, "group.id", topic);
-  Test::conf_set(conf, "enable.auto.commit", "false");
-  Test::conf_set(conf, "enable.partition.eof", "false");
+  Test::conf_set(conf, "enable.auto.commit", "false"); /* now default */
```

Comment:
Configuraiton.md shows the default for `enable.auto.commit` as true

Reply:
Whoops, good spotting!